core/src/main/java/io/stargate/core/metrics/api/Metrics.java

```diff
@@ -36,4 +37,21 @@
    * <p>Note that this registry will warp all the metrics reported via {@link #getRegistry()}.
    */
   MeterRegistry getMeterRegistry();
+
+  /**
+   * Returns set of default micrometer {@link Tags} for the given module.
+   *
+   * @param module module name
+   * @return Tags
+   */
+  Tags tagsWithoutTenant(String module);
+
+  /**
+   * Returns set of default micrometer {@link Tags} for the given module and tenant.
+   *
+   * @param module module name
+   * @param tenant tenant id
+   * @return Tags
+   */
+  Tags tagsWithTenant(String module, String tenant);
```

Comment:
If we have a tag extracting service, I guess this method may not be needed.

Reply:
I was focusing that service only provides a tenant id, that it does nothing about the tags.. I would not mix these.. But yes we could have Tags service and move this stuff there..

Reply:
Removed, will resolve.